docs/getting-started/fundamentals-block-development/the-block-wrapper.md

```diff
@@ -0,0 +1,113 @@
+# The block wrapper
+
+Each block's markup is wrapped by a container HTML tag that needs to have the proper attributes to fully work in the Block Editor and to reflect the proper block's style settings when rendered in the Block Editor and the front end. As developers, when creating a custom block, we need to manually add these attributes to the markup using some of the tools provided by WordPress.
```

Comment:
I find this sentence here a little confusing. It sounds like the markup of the block is wrapped by some markup we cannot control. 

Which was the case with the `apiVersion: 1` but with `apiVersion: 2 & 3` that no longer ist the case and we have the `useBlockProps` hook to get the attributes that need to exist on the wrapper to manually add to our own markup. So a developer has full control over the markup of the block including the wrapper

Reply:
@fabiankaegy I have rephrased that part inspired by the info in your comment.